src/diagnostics/OpenPMDWriter.H

```diff
@@ -14,11 +14,13 @@
 #include <vector>
 
 #ifdef HIPACE_USE_OPENPMD
-#include <openPMD/openPMD.hpp>
+#   include <openPMD/openPMD.hpp>
+#endif
 
 /** \brief Whether the beam, the field data is written, or if it is just flushing the stored data */
 enum struct OpenPMDWriterCallType { beams, fields };
 
+#ifdef HIPACE_USE_OPENPMD
```

Comment:
The matching `#endif` isn't shown here, but it seems to be there nonetheless. Is this just a Github display issue, or is it still missing somehow?

Reply:
Before, the `#ifdef` from line 16 was closed somewhere at the end of the file. Now it is closed at line 18 and the new `#ifdef` from line 23 is closed by the one at the end of the file, that's why github does not show it